bedrock/firefox/views.py

```diff
@@ -497,7 +498,6 @@ class WhatsnewView(L10nTemplateView):
         "firefox/whatsnew/whatsnew-fx133-donation.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx133-donation-eu.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx133-donation-na.html": ["firefox/whatsnew/whatsnew"],
-        "firefox/whatsnew/whatsnew-fx135beta.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx134-us.html": ["firefox/whatsnew/whatsnew"],
```

Comment:
FYI I'm moving the line **up**, between _aurora_ and _release_, instead of at the **end** of release items _(similarly in bundles)_, so the diffs do not conflict with any ongoing release WNP changes underway at the same time, to save for any twelfth hour conflict resolutions.

(Not unlike the existing test & view order, also putting betas between channels, instead of with release.)